test/integration/api/datasets.js

```diff
@@ -2214,10 +2214,7 @@ describe('datasets and entities', () => {
           </h:head>
         </h:html>`;
 
-        it('should NOT autolink on upload new form and simultaneously publish', testService(async (service) => {
```

Comment:
The code was too messy for me to implement this autolinking before, so I had this negative test case lying around.